hw/xilinx/synth/tcl/build_bitstream.tcl

```diff
@@ -25,7 +26,17 @@ source $::env(XILINX_SYNTH_TCL_ROOT)/add_xilinx_sources.tcl
 
 # Load constraints
 import_files -fileset constrs_1 -norecurse $::env(XILINX_ROOT)/synth/constraints/$::env(XILINX_PROJECT_NAME).xdc
-import_files -fileset constrs_1 -norecurse $::env(XILINX_ROOT)/synth/constraints/$::env(XILINX_BOARD).xdc
+import_files -fileset constrs_1 -norecurse $::env(XILINX_ROOT)/synth/constraints/$::env(BOARD).xdc
+
+# Verilog define HPC/EMBEDDED
+if { "$::env(SOC_CONFIG)" == "hpc" } {
```

Comment:
remember to set to zero otherwise

Reply:
With 0 otherwise does not work because 0 is defined and the first `ifdef is activated

Reply:
Can't you do `if` instead of `ifdef` in verilog?

Reply:
No, ``if doesn't exist :(